.env.example

```
@@ -0,0 +1,34 @@
+
+# API Configuration
+GALACTUS_HTTP_PORT=8080
+GALACTUS_GRPC_PORT=9090
+
+# Database Configuration (Future)
+DATABASE_URL=postgresql://user:password@postgres:5432/postgres
+DATABASE_MAX_CONNECTIONS=10
+
+# Monitoring Configuration
+PROMETHEUS_RETENTION_TIME=720h
+GRAFANA_ADMIN_PASSWORD=changeme
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_ADMIN_PASSWORD key should go before the PROMETHEUS_RETENTION_TIME key

(UnorderedKey)